content/en/docs/reference/command-line-tools-reference/kube-proxy.md

```diff
@@ -7,13 +7,15 @@ weight: 30
 ## {{% heading "synopsis" %}}
 
 
-The Kubernetes network proxy runs on each node. This
-reflects services as defined in the Kubernetes API on each node and can do simple
-TCP, UDP, and SCTP stream forwarding or round robin TCP, UDP, and SCTP forwarding across a set of backends.
+A Kubernetes network "proxy" is generally run on all nodes which need to be able to use Kubernete internal services.  It's job is to
```

Comment:
Hi @jayunit100 .
This page is auto-generated using the kube-proxy source code.
Are you able to make this change in the `kubernetes/kubernetes` repository for the `kube-proxy` component?